DoublyLinkedList/header.h

```diff
@@ -0,0 +1,28 @@
+#include <iostream>
+
```

Comment:
header file should be called by a better name. people using your DLL impl will be including this filename in their source. calling it header.h doesnt convey right meaning.

Reply:
ok sir I will rename it